build.sbt

```diff
@@ -40,14 +42,43 @@ val dep = {
   )
 }
 
+lazy val solidityCompileTask = TaskKey[Unit]("solidityCompile", "Compiles solidity contracts")
+
+solidityCompileTask := {
```

Comment:
Nice :+1:, though at some point I think this could be moved to the evm-runner itself